rusk/src/lib/error.rs

```diff
@@ -37,6 +37,8 @@ pub enum Error {
     Canonical(canonical::CanonError),
     /// Stake not found for key.
     StakeNotFound(PublicKey),
+    /// Base58 decoding error.
+    Bs58(bs58::decode::Error),
```

Comment:
If the error was only in the walled and we replace with `Option` we can also avoid this (and all the code related to the formatting / conversion)